karyon3-archaius2/src/main/java/com/netflix/karyon/archaius/admin/ArchaiusResource.java

```diff
@@ -16,7 +16,7 @@
 
 @Singleton
 @AdminService(name="props", index="getAllProperties")
-public class ArchaiusResource {
+final class ArchaiusResource {
```

Comment:
Doesn't this change mean users of karyon have to use the ArchaiusAdminModule and won't have access to inject the implementations themselves in Spring or something?